src/main/assemblies/plugin.xml

```diff
@@ -1,42 +1,49 @@
 <?xml version="1.0"?>
 <assembly>
-  <id>plugin</id>
-  <formats>
-    <format>zip</format>
-  </formats>
-  <includeBaseDirectory>false</includeBaseDirectory>
-  <dependencySets>
-    <dependencySet>
-      <useStrictFiltering>true</useStrictFiltering>
-      <outputDirectory>${file.separator}</outputDirectory>
-      <useProjectArtifact>true</useProjectArtifact>
-      <useTransitiveFiltering>true</useTransitiveFiltering>
-    </dependencySet>
-  </dependencySets>
-  <fileSets>
-    <fileSet>
-      <directory>${basedir}/src/main/assemblies/</directory>
-      <outputDirectory>${file.separator}</outputDirectory>
-      <filtered>false</filtered>
-      <includes>
-        <include>LICENSE</include>
-      </includes>
-    </fileSet>
-    <fileSet>
-      <directory>${project.basedir}</directory>
-      <outputDirectory>${file.separator}</outputDirectory>
-      <includes>
-        <include>tools/**</include>
-        <include>securityconfig/**</include>
-      </includes>
-      <fileMode>0755</fileMode>
-    </fileSet>
-    <fileSet>
-      <outputDirectory>${file.separator}</outputDirectory>
-      <includes>
-        <include>plugin-descriptor.properties</include>
-        <include>plugin-security.policy</include>
-      </includes>
-    </fileSet>
-  </fileSets>
+    <id>plugin</id>
```

Comment:
How that difference in white spaces was introduced? Can you revert it back?

Reply:
There was a cherry-pick conflict for opendistro-0.10. I took changes from the HEAD only. Apart from whitespace and the permission updates I don't see any other differences.